content/ember-5-0-released.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Ember 5 0 Released
+image:
+imageMeta:
+  attribution:
+  attributionLink:
+featured: true
+authors:
+  - jen-weber
```

Comment:
```suggestion
  - jen-weber
  - chriskrycho
```

Reply:
I think this has to be `chris-krycho` to match the filename for your profile in this repo.

Reply:
Doh, I typed it that way out of habit, since I am `chriskrycho` everywhere. Fixing the suggestion now. 😂